charts/back8sup/templates/pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "back8sup.fullname" . }}
+  labels:
+    {{- include "back8sup.labels" . | nindent 4 }}
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
```

Comment:
We should extend the PVC configuration to allow specification of e.g. `storageClassName` and make the storage size request configurable.

Reply:
Yes we should

Reply:
6a5551b8